src/common/helpers/formatPriceRequestVoteData.ts

```diff
@@ -22,6 +22,7 @@ export interface FormattedPriceRequestRounds {
     uniqueClaimersPctOfReveals: string;
     time: number;
     voterRewards: string;
+    ancillaryData: string;
```

Comment:
We need to pass along the ancillary data when formatting so that it can be used later.